agent/functional_tests/tests/functionaltests_unix_test.go

```diff
@@ -1245,3 +1246,42 @@ func TestSSMSecretsEncryptedASMSecrets(t *testing.T) {
 	exitCode, _ := task.ContainerExitcode("ssmsecrets-environment-variables")
 	assert.Equal(t, 42, exitCode, fmt.Sprintf("Expected exit code of 42; got %d", exitCode))
 }
+
+// Note: This functional test requires ECS GPU instance which has atleast 4 GPUs
+// Please use instance like p3.8xlarge for running this test
```

Comment:
we need to confirm this with justin/jake to see if this will work well with the release tests